fish_prompt.fish

```diff
@@ -113,9 +113,10 @@ end
 
 function prompt_finish -d "Close open segments"
   if [ -n $current_bg ]
-    set_color -b normal
+    set_color normal
```

Comment:
Using `-b` with the `normal` keyword doesn't seem to have an effect; both the foreground and background are reset regardless:

![image](https://user-images.githubusercontent.com/2991842/45791595-5f7d6f00-bc58-11e8-95b8-c876850ab324.png)

I figured I'd clean this up while I was in the neighborhood, but I'm happy to keep to just the described fix if it's decided to be too far out-of-scope.

Reply:
Yep, you'll also find that on the man page.